### DISCUSSION TITLE
Resume playback

### DISCUSSION THREAD

**Discussion by Donreeba** [PaulWoitaschek/Voice#1842] [Ideas]
### Overview

Resume playback switch is gone

### How to reproduce

Not in settings anymore

### Version

7.0.7

### Android Version

33

### Device Model

22071212AG

**Comment by PaulWoitaschek** [comment]
I think it didn't do anything for a very long time now.
What do you expect by that switch?

**Comment by Donreeba** [comment]
Was working till now, when i connected my headphones or car stereo it would
start playing without even touching my phone, was very useful at work and
while driving.

**Comment by MartinCorteza** [comment]
I would prefer to have such a switch either. Poweramp has a switch for headphone/bluetooth connection where you can define resume or not if one of these output devices becomes available. For me, "Voice" is always resuming automatically if I connect my phone to my car's bluetooth audio system - a behaviour i'd like to stop.
